packages/webview_flutter/lib/platform_interface.dart

```diff
@@ -167,6 +167,30 @@ abstract class WebViewPlatformController {
     throw UnimplementedError(
         "WebView getTitle is not implemented on the current platform");
   }
+
+  /// Sets the initial scale of page content.
+  ///
+  /// Only supported on Android.
+  ///
+  /// [scaleInPercent] must be between 0 and 100 inclusive.
+  ///
+  /// The behavior for the default initial scale depends on whether the WebView
+  /// supports the "viewport" HTML meta tag, whether the WebView is set to use a
+  /// wide viewport, and whether the WebView loads pages in overview mode, that
+  /// is, zooms out the content to fit on screen by width.
+  ///
+  /// If the content fits into the WebView control by width, then the zoom is
+  /// set to 100%. When loading wide content and the WebView is supporting wide
+  /// content, the content will be zoomed out to be fit by width into the
+  /// WebView control, otherwise not. If initial scale is greater than 0,
+  /// WebView starts with this value as initial scale.
+  ///
+  /// Please note that unlike the scale properties in the viewport meta tag,
+  /// this method doesn't take the screen density into account.
+  Future<void> setInitialScale(int scaleInPercent) {
```

Comment:
If we ever want to support setting the scale to a fraction of a percent we would need a breaking change. Maybe we should start this with a double to be future compatible?

Reply:
tldr: I'll change it to a `double`.

The Android method only uses an `int`. Using a `double` would give the impression that changing from `5.1` to `5.4` would be different values.

Also, wouldn't this be true for almost any method that uses an `int`? Should we only use `int`s in situations where it's impossible to switch to a `double`. (e.g. passing a count value)? If so, we should have that written down in a plugin style guide.

In an instance where the Android method changes, wouldn't it make since to have it as a breaking change since the underlying platform (Java) method is different. 

If you're referring to iOS requiring a `double`, iOS doesn't use a percent, they set a value between a min and max. (e.g.  [minZoomScale](https://developer.apple.com/documentation/uikit/uiscrollview/1619428-minimumzoomscale?language=objc)). I suppose using a `double` would be more accurate than an `int`, so I guess would make sense to use a `double` despite it causing slight confusion for Android.

Reply:
If we think it's reasonable that a value would need to be double in the future (which I think is the case here) we should make it a double as it doesn't cost much and gives us better future compatibility.